easybuild/easyconfigs/p/picard/picard-2.9.2-Java-1.8.0_74.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'JAR'
```

Comment:
@nathanhaigh There's a custom easyblock for `picard`, why not use that?

If needed, we should update it for more recent `picard` versions?